trello/api/features/negative/ListNegative.feature

```diff
@@ -33,3 +32,50 @@ Feature: List
     And Sets a "GET" request to "/lists/ListObject.id"
     And Sends request
     And Should return status code 400
+
+  Scenario: Card with an incorrect id can not be found
+    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/actions"
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
+
+  Scenario: It does not allow to find an non-existent member in card
+    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/members"
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
+
+  Scenario: A card that doesn't exist don't gets a stickers
+    When Sets a "GET" request to "/cards/5e3d75852b8afb5c7c60dc45invalidId/stickers"
+    And Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid id"
+
+  Scenario: Can not Set a soft limit for number of cards in the list with invalid value
+    Given Sets a "POST" request to "/boards/"
+      | key  | value              |
+      | name | newBoardPOSTToList |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "BoardObject"
+    And Sets a "POST" request to "/lists/"
+      | key     | value            |
+      | name    | functionalList   |
+      | idBoard | (BoardObject.id) |
+      | pos     | top              |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "ListObject"
+    When Sets a "PUT" request to "/lists/ListObject.id/softLimit"
+      | key   | value |
+      | value | 5001  |
+    And Sends request
+    Then Should return status code 400
+    And Validates response body with
+      | key     | value                   |
+      | message | invalid value for value |
+      | error   | ERROR                   |
```

Comment:
It would be message not Json, this Scenario would be a @defect

Reply:
here is not necessary to validate as a Json, just as a message. fix this and put as a @defect